src/rez/rezconfig.py

```diff
@@ -499,6 +499,17 @@
 # this value is False.
 allow_unversioned_packages = True
 
+# Defines whether a resolve should immediately fail if any variants have a required package that can't be found.
+# This can be useful to disable if you have packages that aren't available to all users.
+# It is enabled by default. If a variant has requires that cannot be found , it will error immediately rather than
+# trying the other variants.
+# If disabled, it will try other variants before giving up.
+#
+# .. warning::
```

Comment:
@maxnbk and @JeanChristopheMorinPerso , Let me know if this warning here has adequate language.

Reply:
LGTM!